docs/reference/docs/reindex.asciidoc

```diff
@@ -1119,6 +1122,32 @@ POST _reindex
 // TEST[s/otherhost:9200/\${host}/]
 // TEST[s/\.\.\.,/"username": "test_admin", "password": "x-pack-test-password",/]
 
+[[CCS-based-reindex-from-remote]]
+==== CCS based reindex from remote
+
+Reindex supports CCS based reindexing from a remote Elasticsearch cluster:
```

Comment:
nit Would appreciate expanding CCS to Cross Cluster Search at least once in the text (like in this first mention here), and a link to the CCS section (`modules-cross-cluster-search`).

Reply:
[expand and use lowercase](https://github.com/elastic/elasticsearch/pull/117154/commits/34a6708404200dba0aad0371863fb34197779333)